dds/DCPS/DomainParticipantImpl.cpp

```diff
@@ -765,17 +764,17 @@ DDS::ReturnCode_t DomainParticipantImpl::delete_contentfilteredtopic(
     if (DCPS_debug_level > 3) {
       ACE_ERROR((LM_ERROR, ACE_TEXT("(%P|%t) ERROR: ")
         ACE_TEXT("DomainParticipantImpl::delete_contentfilteredtopic, ")
-        ACE_TEXT("can't delete a content-filtered topic \"%C\" ")
+        ACE_TEXT("can't delete contentfiltered topic \"%C\" ")
```

Comment:
I don't understand why this is changing.  The existing log messages are fine, including below when it says "used by a reader".